make_sleuth_folders.py

```diff
@@ -1,61 +1,76 @@
+"""This utility creates directories for the Sleuth files for University
+of London's 'Intro to Programming I' course.
+
+Run the program in the folder where you want to create the directories.
+You can create all the directories, or just the rookie/pro stages."""
+
 import os
 
 # get current directory
-currentPath = os.getcwd()
+CURRENT_PATH = os.getcwd()
+SUB_PATHS = [1, 2, 3, 4]
+
+
+def create_structure():
+    """Create directories and sub-directories required for rookie and
+    pro stages of Sleuth cases."""
 
-print("Files are created in the working directory.")
-print("Which folders do you want to create?" )
+    print("Files are created in the working directory.")
+    print("Which folders do you want to create?")
+
+    # prompt user for input and repeat if invalid
+    answer = None
+    valid_inputs = ["a", "r", "p", "x"]
+    while answer not in valid_inputs:
+        answer = input("(A)ll (R)ookie (P)ro, or e(X)it: ")
+        answer = answer.lower()
+
+    if answer == "a":
+        create_rookie()
+        create_pro()
+        print("Complete")
+    elif answer == "r":
+        create_rookie()
+        print("Complete")
+    elif answer == "p":
+        create_pro()
+        print("Complete")
+    else:
+        exit()
 
-# prompt user for input and repeat if invalid
-answer = 0
-validInputs = ["a", "r", "p", "x"]
-while answer not in validInputs:
-	answer = input("(A)ll (R)ookie (P)ro, or e(X)it: ")
-	answer = answer.lower()
 
-# Create rookie folder function
 def create_rookie():
+    """Create rookie folder function."""
 
-	# create arrays for parent folders and sub folders
-	rookieParent = [101, 201, 202, 301, 302, 303, 401, 402, 403]
-	rookieSub = [1, 2, 3, 4]
+    # create arrays for parent folders and sub folders
+    parent_paths = [101, 201, 202, 301, 302, 303, 401, 402, 403]
 
-	# loop for creating 9 parent folders, each 4 with sub folders
-	for i in range(0, len(rookieParent), 1):
-		rookieParentPath = currentPath + "/%s" % rookieParent[i]
-		os.mkdir(rookieParentPath)
+    # loop for creating 9 parent folders, each 4 with sub folders
+    for directory in parent_paths:
+        new_parent_path = CURRENT_PATH + "/{}".format(directory)
+        os.mkdir(new_parent_path)
+
+        for sub_directory in SUB_PATHS:

```

Comment:
You can see this is reusing the global variable `SUB_PATHS` set at the top of the file. It's usually best to avoid global variables and you could pass arguments to your functions instead.